src/finder.rs

```diff
@@ -188,13 +201,12 @@ impl Default for LinkFinder {
 
 impl<'t> Links<'t> {
     fn new(text: &'t str, url: bool, email: bool, email_domain_must_have_dot: bool) -> Links<'t> {
```

Comment:
I think we should have an option (disabled by default) for the new functionality, something like `protocol_relative_urls(true)`. Normally I wouldn't want to match those kinds of links in plain text.

You can see e.g. GitHub not matching it either: //github.com